framework/tlvf/AutoGenerated/include/tlvf/WSC/eWscVendorExt.h

```diff
@@ -21,9 +21,11 @@ namespace WSC {
 
 enum eWscVendorExtSubelementBssType {
     BACKHAUL_STA = 0x80,
+    FRONTHAUL_BACKHAUL_BSS = 0x60,
```

Comment:
We don't need this - I changed this in https://github.com/prplfoundation/prplMesh/pull/1168, the bss_type is a bitmask so in order to set fronthaul and backhaul types we can simply set:
```
bss_type = eWscVendorExtSubelementBssType::BACKHAUL_BSS | eWscVendorExtSubelementBssType::FRONTHAUL_BSS;
```

Reply:
Relevant commits:
https://github.com/prplfoundation/prplMesh/pull/1168/commits/02bad4695b78f613a84bc8fa79818bd781f801ae
https://github.com/prplfoundation/prplMesh/pull/1168/commits/2dcf4ba4d1c49409494ab9ab0a59b493e2654c39

Reply:
fixed